test/acse-conf/acse-pr.json

```diff
@@ -29,14 +29,6 @@
       "location": "eastus2"
     },
     {
-      "cluster_definition": "disks-managed/swarm-preAttachedDisks-vmas-windows.json",
```

Comment:
managed disks and key vault can be removed for swarm.

But please ensure a test still exists for swarm with the old header.